src/pages/components/About.jsx

```diff
@@ -14,9 +14,7 @@ const About = () => (
     </p>
 
     <p>
-      Think we&apos;ve overlooked something? open up an
-      <a href="https://github.com/cloverinteractive/boilerplate/issues">issue</a> in
-      Github.
+      Think we&apos;ve overlooked something? open up an <a href="https://github.com/cloverinteractive/boilerplate/issues">issue</a> in Github.
```

Comment:
the space is not present when it's in multi line form. so `anissue` was being displayed